explorer/.storybook/preview.js

```diff
@@ -23,6 +24,8 @@ export const parameters = {
 };
 
 const withThemeProvider = (Story, context) => {
+  const currentConfig = useConfig();
+  const theme = getAppTheme(currentConfig.theme);
```

Comment:
By using the useConfig hook and passing this in to `getAppTheme` we are able to access any custom theme defined in `config` for SB, without error.